scripts/get_stats.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass, field
+from glob import glob
+
+@dataclass
+class BenchmarkStats:
+    get: [int] = field(default_factory=list)
+    post: [int] = field(default_factory=list)
+    average_get: int = 0
+    average_post: int = 0
+    average_total: int = 0
+    total_requests: int = 0
+    total_failures: int = 0
+    error_percentage: int = 0
+
+    def __str__(self):
+        return (
+            f'Questionnaire GETs average: {int(self.average_get)}ms\n'
+            f'Questionnaire POSTs average: {int(self.average_post)}ms\n'
+            f'All requests average: {int(self.average_total)}ms\n'
+            f'Total Requests: {int(self.total_requests)}\n'
+            f'Total Failures: {int(self.total_failures)}\n'
```

Comment:
Comma thousand separators would be nice for readability.
```suggestion
            f'Total Requests: {int(self.total_requests):,}\n'
            f'Total Failures: {int(self.total_failures):,}\n'
```